runtime/zkverify/Cargo.toml

```diff
@@ -30,7 +31,7 @@ pallet-bags-list = { workspace = true }
 pallet-bounties = { workspace = true }
 pallet-child-bounties = { workspace = true }
 pallet-balances = { workspace = true }
-frame-support = { workspace = true }
+frame-support = { workspace = true, features = ["tuples-96"] }
```

Comment:
WOW.... Really identity need it?  I'm wondered to know how the compile time increase with this feature active.

Reply:
Haven't noticed much increase, btw the problem is the construct_runtime macro taking lot of stuff currently